apps/browser/qml/pages/components/Overlay.qml

```diff
@@ -476,10 +493,12 @@ Shared.Background {
                     overlayAnimator.showOverlay()
                 }
                 onShareActivePage: {
-                    pageStack.animatorPush("Sailfish.WebView.Popups.ShareLinkPage", {
-                                               "link": webView.url,
-                                               "linkTitle": webView.title
-                                           })
+                    var content = {
+                        "type": "text/x-url",
+                        "linkTitle": webView.title,
+                        "status": webView.url,
+                    }
+                    _shareContent(content, content.type)
```

Comment:
the old page had a header: "Share link" etc.  Should that (translated) UI context hint be provided as some content data, so that the sharing dialog can display it?  Or is that already handled somehow?

Reply:
I checked this with Martin and indeed the header should be kept. I've restored this now, in this MR.